pnpm-lock.yaml

```diff
@@ -1,31 +1,7 @@
-lockfileVersion: 5.3
```

Comment:
@sw-yx you could probably drop this file from the commit. It looks like you might be using an older version of pnpm because it downgraded to `lockfileVersion: 5.2`

Reply:
ah sorry about that, didn't realize